src/main/java/org/opentripplanner/transit/api/request/TripOnServiceDateRequestBuilder.java

```diff
@@ -0,0 +1,66 @@
+package org.opentripplanner.transit.api.request;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.timetable.TripAlteration;
+
+public class TripOnServiceDateRequestBuilder {
+
+  private List<FeedScopedId> authorities = List.of();
+  private List<FeedScopedId> lines = List.of();
+  private List<FeedScopedId> serviceJourneys = List.of();
+  private List<FeedScopedId> replacementFor = List.of();
+  private List<String> privateCodes = List.of();
+  private List<TripAlteration> alterations = List.of();
+  private List<LocalDate> operatingDays;
+
+  protected TripOnServiceDateRequestBuilder() {}
+
+  public TripOnServiceDateRequestBuilder withOperatingDays(List<LocalDate> operatingDays) {
+    this.operatingDays = operatingDays;
```

Comment:
No nullcheck is needed here - this is now safe. You may as well clear a field using null.